source/include/core_mqtt.h

```diff
@@ -598,21 +598,23 @@ MQTTStatus_t MQTT_Disconnect( MQTTContext_t * pContext );
  * alive.
  *
  * @note Passing a timeout value of 0 will run the loop for a single iteration.
- * If a dummy #MQTTGetCurrentTimeFunc_t was passed to #MQTT_Init, then the timeout
```

Comment:
We should just say that it is not supported with dummy timer function and reference doxygen for timer function ( to explain what we mean by dummy)

Reply:
The presence of `#MQTTGetCurrentTimeFunc_t` adds the reference to the documentation of the timer function

Reply:
>We should just say that it is not supported with dummy timer function 
Only the keep-alive part of the function is not supported with a dummy timer function. The rest of the functionality of receiving acks and incoming PUBLISH is supported